config/main.py

```diff
@@ -3427,6 +3427,36 @@ def speed(ctx, interface_name, interface_speed, verbose):
         command += " -vv"
     clicommon.run_command(command, display_cmd=verbose)
 
+#
+# 'link-training' subcommand
+#
+
+@interface.command()
+@click.pass_context
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.argument('mode', metavar='<mode>', required=True, type=click.Choice(["on", "off"]))
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def link_training(ctx, interface_name, mode, verbose):
+    """Set interface auto negotiation mode"""
```

Comment:
Typo. Should be link training instead of auto negotiation

Reply:
Thanks